source/plugins/idea.rst

```diff
@@ -1,10 +1,36 @@
-=============
+=======
 IntelliJ IDEA
-=============
+=======
 
+.. tip::
+    IntelliJ is recomended to use by the Sponge team, as it is a very smart IDE for beginners and advanced users.
```

Comment:
I'm not sure this kind of endorsement should be in the documentation, it's up to end users to choose which IDE they prefer.